README.md

```diff
@@ -30,6 +30,14 @@ native layer.
 - You can just run the following command to test.
   - ```./bacardi test```
 
+## Lint
+- You can just run the following command to lint.
+  - ```./bacardi lint```
+ 
+## Cherry-pick a pull request
```

Comment:
I'm not sure if it's good to write this section here.
We might have to write a new separated documentation.

Reply:
I think so too

Reply:
@romandev, @yjaeseok, this pr is based on the content in the issue but I also thought of adding pr command in README.md might need to be in another file.

Reply:
@nadongguri So i think this pr could be closed, and then we can make pr for another file.